src/NServiceBus.PersistenceTests/IPersistenceTestsConfiguration.cs

```diff
@@ -47,9 +47,9 @@ public interface IPersistenceTestsConfiguration
     // Consumers of this source package have to implement the remaining properties via partial class to configure the tests infrastructure.
     public partial class PersistenceTestsConfiguration : IPersistenceTestsConfiguration
     {
-        public PersistenceTestsConfiguration(TestVariant variant, TimeSpan? sessionTimeout = null)
+        public PersistenceTestsConfiguration(TestVariant variant)
```

Comment:
I could also implement this non-breaking by reintroducing the `sessionTimeout` and then in the constructor let that take precedence over the TestVariant but I didn't feel this to be necessary and potentially weird because why would the constructor win over the variant or vise-versa